### PR TITLE
Release `v3.3.3`

### PR DESCRIPTION
**Pull request by paulhazen**
These changes represent the upcoming release (`v3.3.3`)

#EOS-2115

### CODE REVIEW

File: com.playeveryware.eos/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,33 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# Changelog
+## [3.3.3] - 2024-09-19
+
+### Features
```
Comment (paulhazen) [inline com.playeveryware.eos/CHANGELOG.md]: This should be changed to "Added."

File: com.playeveryware.eos/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,33 @@
...
+- **Unit Testing Enhancements**:
+  - Added new unit test documentation and expanded test coverage across the project. (#914, #905, #893, #fdd07613, #0f55d1ec, #902)
+
+- **Documentation Updates**:
+  - Improved documentation clarity and organization, including replacing markdown files in the root of the repository with symbolic links to the package files. (#914, #f9d3615a, #8685cd73, #f077cec1)
+
+### Fixes
```
Comment (paulhazen) [inline com.playeveryware.eos/CHANGELOG.md]: This should be changed to "Fixed."

File: com.playeveryware.eos/CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,33 @@
...
+- **Android Build Tools**:
+  - Updated the `GetBuildTools` function to accommodate Android Build Tool versioning in Unity 2022.3.44f1 LTS. (#8ed407cf, #bc075e7e)
+
+- **Namespace and Manager Refactoring**:
+  - Refactored managers into services, removed outdated manager references, and updated namespaces. (#2060ce08, #e83deb7f, #57b7b8b6, #561534d5)
+
+- **Miscellaneous Fixes**:
+  - Removed extraneous files like `.editorconfig` and unreferenced classes. Fixed small issues like grammar and unused code. (#116f51c7, #84b67848, #adfb2877, #e3c7262a, #c799fcc1)
+
+### Tests
```
Comment (paulhazen) [inline com.playeveryware.eos/CHANGELOG.md]: This section should be removed, and it's contents should be moved under the "Added" section.

Comment (paulhazen) [reply com.playeveryware.eos/CHANGELOG.md]: Done.

File: com.playeveryware.eos/CHANGELOG.md (hunk 2 above)
Comment (paulhazen) [reply com.playeveryware.eos/CHANGELOG.md]: Done.

File: com.playeveryware.eos/CHANGELOG.md (hunk 1 above)
Comment (paulhazen) [reply com.playeveryware.eos/CHANGELOG.md]: Done.

File: com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs [hunk 4]
```diff
@@ -513,7 +513,7 @@ public static async Task<string> ReadAllTextAsync(string path)
         {
 #if UNITY_ANDROID && !UNITY_EDITOR
             // On Android, use a custom helper to read the file synchronously
-            return await Task.FromResult(AndroidFileIOHelper.ReadAllText(path));
+            return await AndroidFileIOHelper.ReadAllText(path);
```
Comment (WispyMouse) [inline com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs:516]: Is this actually running synchronously? This looks like a normal async Task thing to me, maybe comment needs changing or we need to make sure to do this synchronously?

**Review by WispyMouse — APPROVED** [review]

File: com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs (hunk 4 above)
Comment (paulhazen) [reply com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs:516]: Comment needs changing - I've updated it to reflect that.

**Review by arthur740212 — APPROVED** [review]

**Pull request merged**